CMakeLists.txt

```diff
@@ -29,38 +33,54 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_AUTOUIC ON)
 
-if(CMAKE_VERSION VERSION_LESS "3.7.0")
-    set(CMAKE_INCLUDE_CURRENT_DIR ON)
-endif()
-
-set(RELEASE_NAME "Makou_Reactor")
-set(RELEASE_VERSION "1.8.2")
-set(GUI_TARGET "${RELEASE_NAME}")
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-find_package(Qt5 COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
+find_package(QT NAMES Qt5 COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
 find_package(ff7tk REQUIRED NO_MODULE COMPONENTS ff7tk ff7tkUtils)
 find_package(ZLIB REQUIRED)
 
+file(GLOB_RECURSE TS_FILES "${RELEASE_NAME}_*.ts")
 file(GLOB_RECURSE PROJECT_SOURCES "*.cpp" "*.h")
-
-# Translations
-file(GLOB_RECURSE translation_files "${RELEASE_NAME}_*.ts")
-set_source_files_properties(${translation_files} PROPERTIES OUTPUT_LOCATION "${CMAKE_INSTALL_FULL_BINDIR}")
-qt5_add_translation(QM_FILES ${translation_files})
-
-if(WIN32)
-    set (FF7TK_DEPENDS
-        ${ff7tk_DIR}/../../../bin/ff7tk.dll
-        ${ff7tk_DIR}/../../../bin/ff7tkWidgets.dll
-    )
-endif()
+set(RESOURCES "src/qt/${RELEASE_NAME}.qrc")
 
 # GUI
-add_executable(${GUI_TARGET} ${PROJECT_SOURCES} ${QM_FILES} src/qt/${RELEASE_NAME}.qrc src/qt/${RELEASE_NAME}.rc)
+add_executable(${GUI_TARGET} ${PROJECT_SOURCES} ${QM_FILES} ${RESOURCES} src/qt/${RELEASE_NAME}.rc)
```

Comment:
You should add Both WIN32 and MACOSX_BUNDLE between ${GUI_TARGET} ${PROJECT_SOURCES}
Tldr this makes sure on win32 you build a gui application and on mac os you create a proper bundle (i.e .app file)